lib/models/repository-states/present.js

```diff
@@ -144,6 +171,46 @@ export default class Present extends State {
     }
   }
 
+  isCommitMessageClean() {
+    if (this.commitMessage.trim() === '') {
+      return true;
+    } else if (this.commitMessageTemplate) {
+      return this.commitMessage === this.commitMessageTemplate;
+    }
+    return false;
+  }
+
+  async updateCommitMessageAfterFileSystemChange(events) {
+    for (let i = 0; i < events.length; i++) {
+      const event = events[i];
+      const endsWith = (...segments) => event.path.endsWith(path.join(...segments));
```

Comment:
Might be worth extracting this to `helpers.js`?

It _is_ a one-liner, but eh.

Reply:
sure!  Good suggestion.  Extracted.